test/MicrosoftTeams.spec.ts

```diff
@@ -169,20 +181,63 @@ describe("MicrosoftTeams", () => {
     let getContextMessage = findMessageByFunc("getContext");
     expect(getContextMessage).not.toBeNull();
 
-    processMessage({
-      origin: "https://some-malicious-site.com/",
-      source: microsoftTeams._window.parent,
-      data: {
-        id: getContextMessage.id,
-        args: [
-          {
-            groupId: "someMaliciousValue"
-          }
-        ]
-      } as MessageResponse
-    } as MessageEvent);
+    unSupportedDomains.forEach(unSupportedDomain => {
+      processMessage({
+        origin: unSupportedDomain,
+        source: microsoftTeams._window.parent,
+        data: {
+          id: getContextMessage.id,
+          args: [
+            {
+              groupId: "someMaliciousValue"
+            }
+          ]
+        } as MessageResponse
+      } as MessageEvent);
+
+      expect(callbackCalled).toBe(false);
+    });
+  });
+
+  it("should allow messages from supported domains", () => {
+    initializeWithContext("content");
+    const supportedDomains = [
+      "https://teams.microsoft.com",
+      "https://teams.microsoft.us",
+      "https://int.teams.microsoft.com",
+      "https://devspaces.skype.com",
+      "http://dev.local",
+      "https://microsoft.sharepoint.com",
+      "https://a.b.sharepoint.com",
```

Comment:
Do these pass? I think "*" in the pattern matches *exactly* one subdomain.

Reply:
Looks like the way i have setup Unit test is wrong. i am working on fixing it.